tools/bpf2c/bpf2c.vcxproj

```diff
@@ -132,6 +135,9 @@
     <PreBuildEvent>
       <Command>powershell -NonInteractive -ExecutionPolicy Unrestricted $(SolutionDir)scripts\Process-File.ps1 -InputFile Convert-BpfToNative.ps1.template -OutputFile $(OutDir)Convert-BpfToNative.ps1 -ConfigFile replacements.json</Command>
     </PreBuildEvent>
+    <ResourceCompile>
```

Comment:
Consider putting the `<ResourceCompile>` additional include directories into `Directory.Build.props` so it happens for everything in the repo including things that get added in the future, rather than putting in each project individually?